### DISCUSSION TITLE
The Tech We Want event

### DISCUSSION THREAD

**Discussion by Parijat-03** [semanticClimate/ipcc-report-cataloguing#14] [General]
A series of demonstrations for OKF , [The Tech We Want Online Summit](https://okfn.org/en/events/the-tech-we-want-online-summit/) - 17 October 2024 - hosted by the Open Knowledge Foundation (OKFN), is a one-day online event where experts in public interest technologies come together to discuss practical ways to create software that is helpful, easy to use, durable, and solves real-world problems.

**Comment by Parijat-03** [comment]
# amilib
## **OVERVIEW**
1. ```amilib``` is a Python library designed for document processing, and dictionary creation. It has tools for finding, cleaning, converting, searching, republishing legacy documents (PDF, PNG, etc.).

2. Building dictionaries from wordlist : The dictionaries are based on Wikimedia and have words from the wordlist created from the IPCC chapters

3. Annotating IPCC chapters : The annotated IPCC chapters have highlighted words that are linked to the dictionary


# pygetpapers
This tool is used to search literatures from the open access scientific journals, getting the numbers of papers available and downloads the pdf also, if accessible. It allows access to the literatures from EPMC.
1. Demonstrating pygetpapers.

# Dictionary
Dictionaries are built from the wordlists and are structured collection of the terms and their definitions in a HTML format. It is enhanced with Wikimedia.

**Comment by mrchristian** [comment]
### Preparing the demo of IPCC Glossary for OKF

#### Tasks

- [ ] Check on source of IPCC Glossary as as supplied by #semanticClimate (sC)
- [ ] Source needed checking against IPCC Glossary source as in sC version numerals were missing
- [ ] Make sure IPCC Glossary is in Semantic Wikibase (SWB) install - https://climatekg.semanticclimate.net/index.php?title=IPCC_Begriffe (in German, requires registration)
- [ ] Make SWB instance multilingual (PMR - at what level? probably only keywords - or do you mean add English to the German-only?)
- [ ] Open content to public
- [ ] Have IPCC Glossary output to own GitHub Pages site - will all terms
- [ ] Suppprt glossary data export for sC
- [ ] Make data model for IPCC Glossary - currently data model is below minimal needed

**Comment by petermr** [comment]
# our tech

A list of the things we want to show to OKF.

## pygetpapers AND tabular ouput

- [ ] NOTE: Ayush uses dataframes and `datables` in `pygetpapers` - I didn't know that. Make s it easier.

### add hyperlinks to cells
(prototyped by PMR)

### reduce number of columns
(prototyped by PMR)

### lookup keywords in titles and abstracts and add hyperlinks
Requires a phrase extractor. Then use wikimedia kookup

## amilib

**Comment by mrchristian** [comment]
Question:

What DH & Repository tooling is out there for working on a Corpus: Corpus Management, Packaging, Semtification, carry out Data Analysis and producing research outputs. 

https://github.com/semanticClimate/JEP-article/issues/1

Also posted to NFDI4Culture: https://tickets.nfdi4culture.de/work_packages/9750/activity

What DH & Repository tooling (software) is out there for working on a Corpus: Corpus Management, Packaging, Semtification, carry out Data Analysis and producing research outputs.

The reason for asking the question is that for an individual publication how do we make a publication usable, compatible with standards used in existing systems for corpus packaging and data analysis.

The kinds of tasks, functions, capabilities being looked at are:

- Collect corpus into one file system
- Package corpus with an inventory
- Corpus conversion to open standard format, interoperable standard, have validation of open standard format
- Corpus versioning and forking
- Semantification: Annotate with Names Entifty Recognition,
- Semantic concept annotation
- Enable NLP anaysis: Word frequency
- Enable syntactic and syntactic/semantic markup
- Enable TDM
- Research outputs: Allow for analysis of finding and results outputted as data and corpus copy if needed as Open Science being compatible
- Reporting on Corpus: Bibliometric, Presenting Knowledge and ideas, statitics to back findings, etc
